### ISSUE TITLE
Quality Focal Loss become infinitive

### ISSUE THREAD

**Issue by xuanthuyvo** [implus/GFocalV2#4]
Thank you for your great work! I have a problem when I tried to run your model on my single computer. The quality focal loss is so big up to value 2000-3000. Do you have any idea to solve this problem?
Thank you so much!

**Comment by liushuai839** [comment]
Hello, I want to see the source code of GFLv2. But I cannot find it. And this project seems the same as GFLv1, could you tell me  where the coresponding code is?  Thanks!

**Comment by implus** [comment]
> Thank you for your great work! I have a problem when I tried to run your model on my single computer. The quality focal loss is so big up to value 2000-3000. Do you have any idea to solve this problem?
> Thank you so much!

We default to use 8 GPUs with a total batch size of 16. If you use smaller batch size in your single computer, I suggest you use smaller learning rate which may help~

**Comment by xuanthuyvo** [comment]
Thank you for your help. I will try to train on multiple GPUs and set suitable learning rate.

**Comment by xuanthuyvo** [comment]
> Hello, I want to see the source code of GFLv2. But I cannot find it. And this project seems the same as GFLv1, could you tell me where the coresponding code is? Thanks!

You can get the code here: https://github.com/implus/GFocalV2/blob/master/mmdet/models/dense_heads/gfocal_head.py

**Comment by implus** [comment]
config的 GFocalHead 里面
loss_cls=dict(
type='QualityFocalLoss',
use_sigmoid=False,
beta=2.0,
loss_weight=1.0),

注意use_sigmoid=False

很多对应的代码有一些变化，请注意一下~
